pkg/kubelet/apis/config/types.go

```diff
@@ -397,6 +397,24 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration
+	// podPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based on their associated priority class value. Requires the GracefulNodeShutdown feature gate to be enabled.
+	// Each entry in the array represents the graceful shutdown time a pod with a priority class value that
+	// lies in the range of that value and the next higher entry in the list when the node is shutting down.
+	// For example:
+	// |Pod priority class value|Shutdown period|
+	// |------------------------|---------------|
+	// | 100000                 |10 seconds     |
+	// | 10000                  |180 seconds    |
+	// | 1000                   |120 seconds    |
+	// | 0                      |60 seconds     |
+	// With the above configuration set, any pod with priority value >= 100000 will get
+	// 10 seconds to stop, any pod with value >= 10000 and < 100000 will get 180
+	// seconds to stop, any pod with value >= 1000 and < 10000 will get 120 seconds to stop.
+	// All other pods will get 60 seconds to stop.
```

Comment:
I would expect text describing this can't be used with ShutdownGracePeriodCriticalSeconds also specified (only one of the two features can be used together).

Reply:
This comment is still outstanding and needs to be addressed.